### PR TITLE
Reader: Work in changes to lists for upcoming refresh

### LINKED ISSUE #9195: Tweaks to Lists in Sidebar
For the Lists element in the Sidebar:

If the user has 0 lists, remove the Lists element.

tbt11

If the user has 1 or more lists, retain the Lists element, but remove the ADD and MANAGE links.

tbt2

This can happen anytime before the refresh launch.

### PR DESCRIPTION
**Pull request by blowery**
- Turn off lists sidebar for folks who do not have any lists
- Turn off add / manage links in general

Originally #9269

Fixes #9195

### CODE REVIEW

**Comment by blowery** [comment]
after, no lists 
following_ _reader_ _wordpress_com

**Comment by blowery** [comment]
after, with existing list subs
_2__following_ _reader_ _wordpress_com

**Comment by blowery** [comment]
before, with lists
following_ _reader_ _wordpress_com

before, no lists
following_ _reader_ _wordpress_com

File: client/reader/sidebar/expandable-add-form.jsx [hunk 1]
```diff
@@ -1,45 +1,35 @@
...
+		this.toggleAdd = this.toggleAdd.bind( this );
+		this.handleAddKeyDown = this.handleAddKeyDown.bind( this );
+		this.state = {
+			isAdding: false,
 		};
-	},
-
+	}
+	
 	toggleAdd() {
```
Comment (omarjackman) [inline client/reader/sidebar/expandable-add-form.jsx]: Not a big deal at all, tomaytoes/tomahtoes, but we can do `toggleAdd = () =>` now to get the binding to `this`

**Comment by samouri** [comment]
Tested:
- [x] add/manage disappears for user with 1+ lists, but the lists remains
- [x] all of lists disappear for users with 0 lists.
- [x] direct urls to list management still work

🚢

**Pull request merged**